src/containers/User/UserArea.tsx

```diff
@@ -47,6 +48,9 @@ const UserArea: React.FC = () => {
               {user}
             </Button>
           ))}
+        <Button basic color="blue" onClick={logOut}>
+          Log out
+        </Button>
```

Comment:
This made me think we need a issue just to work on the UserArea layout. Not a big priority but needs to be done. Do you mind creating this issue please? I'm planning to go through all issues and epics on the weekend so we can try visualising our milestones and what need to be done in the next future.

Reply:
👍 #148